tools/setup_helpers/cudnn.py

```diff
@@ -1,97 +0,0 @@
-import os
-import glob
-
-from .env import IS_WINDOWS, IS_CONDA, CONDA_DIR, check_negative_env_flag, gather_paths, lib_paths_from_base
-from .cuda import USE_CUDA, CUDA_HOME
-
-
-USE_CUDNN = False
-CUDNN_LIB_DIR = None
-CUDNN_INCLUDE_DIR = None
-CUDNN_LIBRARY = None
-WITH_STATIC_CUDNN = os.getenv("USE_STATIC_CUDNN")
-
-if USE_CUDA and not check_negative_env_flag('USE_CUDNN'):
-    lib_paths = list(filter(bool, [
-        os.getenv('CUDNN_LIB_DIR')
-    ] + lib_paths_from_base(CUDA_HOME) + [
-        '/usr/lib/x86_64-linux-gnu/',
-        '/usr/lib/powerpc64le-linux-gnu/',
-        '/usr/lib/aarch64-linux-gnu/',
-        '/usr/lib/',
-    ] + gather_paths([
-        'LIBRARY_PATH',
-    ]) + gather_paths([
-        'LD_LIBRARY_PATH',
-    ])))
-    include_paths = list(filter(bool, [
-        os.getenv('CUDNN_INCLUDE_DIR'),
-        os.path.join(CUDA_HOME, 'include'),
-        '/usr/include/',
-    ] + gather_paths([
-        'CPATH',
-        'C_INCLUDE_PATH',
-        'CPLUS_INCLUDE_PATH',
-    ])))
-    # Add CUDA related dirs to candidate list
-    if IS_CONDA:
-        lib_paths.append(os.path.join(CONDA_DIR, 'lib'))
-        include_paths.append(os.path.join(CONDA_DIR, 'include'))
```

Comment:
An equivalent to these two lines are at https://github.com/pytorch/pytorch/blob/0b1fee0819c82736cef387c3f02b7aa4c75419aa/tools/setup_helpers/cmake.py#L210